buku.py

```diff
@@ -1492,6 +1492,31 @@ def get_page_title(resp):
         return htmlparser.parsed_title
 
 
+def create_poolmanager(url):
```

Comment:
> And let's support only https proxies

Would you mind to elaborate on the reason for only supporting https proxies? I believe that those variables are not meaning that the traffic between our host and the proxy is cryptographed or not. They only enable an app to use differente proxy servers for different kinds of traffic.

I agree that the checks are excessive, though.